lib/app/components/organisms/message_bar.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+class MessageBar extends StatelessWidget {
+  const MessageBar({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: const Alignment(0, -1),
+      child: Container(
+        width: double.infinity,
+        height: 70,
+        decoration: BoxDecoration(
+          color: Colors.white,
+          border: Border.all(
+            color: Colors.black,
+          ),
+        ),
+        child: Row(
+          children: [
+            IconButton(
+              //TODO: カメラ機能は別途実装
+              onPressed: () {},
+              icon: const Icon(
+                Icons.camera_alt_rounded,
```

Comment:
```suggestion
                Icons.camera_alt_outlined,
```
すいません、Miroに反映できないですが、アイコンが色が塗られているものより、
outlineにした方がダークモードみたいに背景色変更したときに対応しやすいので、
アイコンはoutlineでお願いします🙇

Reply:
ご指摘のとおり修正しました。